# recovery behavior
hz: 10
sim_time_samples: 10
vel_samples: 3
move_time: 2.0
min_vel_x: -0.2
max_vel_theta: 0.4

# footprint
front_side_length: 0.1
rear_side_length: 0.1
left_side_length: 0.1
right_side_length: 0.1
